app.py

```diff
@@ -69,6 +70,15 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
     domain_name=domain.stac_domain_name,
 )
 
+stac_browser = StacBrowser(
+    veda_stack,
+    "stac-browser",
+    github_repo_tag="v3.1.0", # hard coded to the latest for now. 
+    stac_catalog_url=stac_api.stac_api_url, # using the non-custom-domain for now. 
```

Comment:
I think we might need to add the `api/stac` path prefix here

Reply:
Added it directly in the stac-api `construct`